### PR TITLE
Netlify case study.md

### PR DESCRIPTION
**Pull request by Sugal99**

### CODE REVIEW

File: src/content/docs/webdev-hosting/Netlify/netlify-case-study.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- Complexity for Large-Scale Applications: Since Netlify is more suited for small applications, you may find it lack more advanced features and options for large scale
+  applications. Beacuse of this organzations may choose other options for hosting instead.
+
+- Limited Backend: As stated earlier Netlify relies on serverless functions, because of this it may limit amount of backend customization and control available to developers.
+
+- Third-Party Services: Third-Party Services is a big part of netlify, it relies on services like AWS and and CDN services. The big risk with this is that this may introduce dependencies which then makes uptime and reliability affected.
+
+## Market Comparison
+
+If compared to a more flexible and advanced hosting website like Heroku, I would say that Netlify excels and shines the most when it comes to hosting static websites and fast deployment while offering great features such as Git Intergrations, Forms Handling and serverless functions.
```
Comment (sulenchy) [inline src/content/docs/webdev-hosting/Netlify/netlify-case-study.md:73]: Consider rephrasing this using a third-person form. As it is, it is not objective and universal enough.
```suggestion
When compared to a more flexible and advanced hosting website like Heroku, Netlify excels and shines the most in hosting static websites and fast deployment while offering great features such as Git integrations, forms handling, and serverless functions.
```

File: src/content/docs/webdev-hosting/Netlify/netlify-case-study.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+While on the other hand we have heroku, which is pretty much a lot like Netlify besides the fact that its more advanced and more flexible. Heroku provides good support for applications that takes use of full-stack web applications with server-side logic and databases.
```
Comment (sulenchy) [inline src/content/docs/webdev-hosting/Netlify/netlify-case-study.md:75]: Same thing here. Fix the tone and grammar. I recommend you use tools like quillbot.com

File: src/content/docs/webdev-hosting/Netlify/netlify-case-study.md (hunk 1 above)
Comment (Sugal99) [reply src/content/docs/webdev-hosting/Netlify/netlify-case-study.md]: > Consider rephrasing this using a third-person form. As it is, it is not objective and universal enough.

This is the fix that i pushed, is this fine? https://github.com/Sugal99/development-platforms/commit/39db174a6ce78ad04a0af12f31ee85644e50819c

**Review by sulenchy — APPROVED** [review]
Thanks for implementing the feedback

**Pull request merged**